dev/cosri/docker-compose.yaml

```diff
@@ -0,0 +1,51 @@
+version: "3.7"
+services:
+  frontend:
+    build:
+      context: https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY.git
+      dockerfile: Dockerfile.production
+
+    environment:
+      # confidential backend API endpoint
+      # NB: domain is required for cookies to be set
+      REACT_APP_CONF_API_URL: "https://backend.${BASE_DOMAIN:-localtest.me}"
+
+      
+    labels:
+      - "traefik.enable=true"
+
+      - "traefik.http.routers.frontend.rule=Host(`frontend.${BASE_DOMAIN:-localtest.me}`)"
+      - "traefik.http.routers.frontend.entrypoints=websecure"
+      - "traefik.http.routers.frontend.tls.certresolver=myresolver"
+    networks:
+     - ingress-network
+
+  backend:
+    image: "uwcirg/sof-api-wrapper:develop"
+
+    environment:
+      SECRET_KEY: qiLYeYMJf8i7y0xU
```

Comment:
why aren't we pulling secrets from outside the repository?  (Guessing that will be a subsequent PR, just want to get it running end to end - I'm fine with that, for now)

Reply:
we'll probably use `.env` or a `env_file` directive to keep eventually (certainly for prod), but having it inline requires less configuration